### ISSUE TITLE
Veinminer + Actually Adiditions 

### ISSUE THREAD

**Issue by mistahoward** [DNSTechpack/DNS10-MC1.10.2#81]
* **Type of issue?**
  * BUG
  
* **DNS Version?**
  * 10.3.0.4
  

* **What should happen?** 
  * Obsidian Axe (Actually Additions) should vein mine trees.
  

* **What actually happens?**
  * No vein miner.

**Comment by IanMoroney** [comment]
alternatively you could use your hand?

**Comment by robaimes** [comment]
You can submit a pull request to add in this functionality, it would alleviate some of the small tasks for us and its a great help.

If not, I will try to do this for next time.

**Comment by Lazarix** [comment]
Go into ATLauncher\Instances\DNSTechpack\config\veinminer\general.config
set the override for allBlocks to be True. Like this:

overrides {
    # Mine all blocks with all registered tools that can harvest blocks.
    B:override.allBlocks=true

The obsidian axe will then work (Tested and confirmed)

**Comment by robaimes** [comment]
This is **not** a solution. This will not only add this functionality, it will also add the ability to veinmine *every* block if using the specified tool. We turned this off because of performance issues.

As the button is set to sneak, you can quite easily end up accidentally mining every single bit of stone in the immediate radius around you, and you're probably gonna have a bad time.

**Comment by definablefan4** [comment]
I agree with Omi. It is set to this on skyfactory and I can not tell you how many times I have accidentally veinmined my whole base.

**Comment by Lazarix** [comment]
<disclaimer>
Use with caution

**Comment by Lazarix** [comment]
I found the problem. will submit a PR.

**Comment by Lazarix** [comment]
PR done: https://github.com/DNSTechpack/DNS10-MC1.10.2/pull/82
